.github/workflows/workflow.yml

```diff
@@ -86,6 +86,8 @@ jobs:
         uses: codota/toolchain@00a8bf2bdcfe93aefd70422d3dec07337959d3a4
         with:
           profile: minimal
+      - name: Check formatting
```

Comment:
i would move it to a separate job, mby even separate workflow altogether as you may want to see results of the build regardless of your formatting